test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/ReadWrite/Updating/Resources/ReplaceToManyRelationshipTests.cs

```diff
@@ -0,0 +1,130 @@
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using JsonApiDotNetCore.Serialization.Objects;
+using Xunit;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.ReadWrite.Updating.Resources
+{
+    public sealed class ReplaceToManyRelationshipTests
+        : IClassFixture<IntegrationTestContext<TestableStartup>>
+    {
+        private readonly IntegrationTestContext<TestableStartup> _testContext;
+
+        public ReplaceToManyRelationshipTests(IntegrationTestContext<TestableStartup> testContext)
+        {
+            _testContext = testContext;
+        }
+        
+        [Fact]
+        public async Task Cannot_replace_HasMany_relationship()
+        {
+            // Arrange
+            var existingWorkItem = new WorkItem();
```

Comment:
Aside from earlier remarks on faker usage, which applies to multiple files which I'm not all going to highlight, the fact that only a workitem (without relationships) is being created makes that the test name no longer matches the scenario. Instead of replacement, it has become assignment. I think it would be preferable to stick with the original intent.